app/javascript/Pages/components/Footer/Footer.css

```diff
@@ -0,0 +1,90 @@
+.footer-container {
+  background-color: #252636;
+  padding: 4rem 0 3rem 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-self: center;
+}
+
+.footer-links {
+  width: 100%;
+  max-width: 1000px;
+  display: flex;
+  justify-content: center;
+  margin: 60px auto 0 auto;
+}
+
+.footer-link-wrapper {
+  display: flex;
+}
+
+.footer-link-items {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin: 16px;
+  text-align: left;
+  box-sizing: border-box;
+}
+
+.footer-link-items a {
```

Comment:
**suggestion (blocking)**: prefer using a class instead of applying the styles to html elements (you might need to use the same style for something that is not an a)